src/main/groovy/de.flachnetz.golang/GoPlugin.groovy

```diff
@@ -220,22 +233,34 @@ class GoPlugin implements Plugin<Project> {
                     commandLine go, "build", "-a", "-ldflags",
                             "-X=main.Version=${project.version} -X=main.GitHash=${gitHash} -X=main.BuildDate=${DateTimeFormatter.ISO_DATE_TIME.format(LocalDateTime.now())}",
                             "-o", new File(project.buildDir, config.binaryName)
-                    workingDir canonicalImportFile
+
+                    workingDir projectCanonicalImportFile
                     environment defaultEnvironmentVariables
+                    environment "CGO_ENABLED", config.cgoEnabled ? 1 : 0
                 }
 
-                project.task("buildStaticForLinux", type: Exec, dependsOn: ["dependencies"]) {
+                project.task("buildForLinux", type: Exec, dependsOn: "dependencies") {
```

Comment:
cgo is controlled by cgoEnabled. It wouldnt be fair to call it "static" and then enable cgo - which makes it non-static.